src/dynamic_whitelist_bridge.cpp

```diff
@@ -762,25 +758,23 @@ int main(int argc, char *argv[]) {
         // explicitly avoid topics with more than one type
         if (topic_and_types.second.size() > 1) {
           if (already_ignored_topics.count(topic_name) == 0) {
-            std::string types = "";
-            for (auto type : topic_and_types.second) {
-              types += type + ", ";
+            std::stringstream types;
+            for (const auto& type : topic_and_types.second) {
+              types << type << ", ";
             }
-            fprintf(
-                    stderr,
-                    "warning: ignoring topic '%s', which has more than one type: [%s]\n",
+            RCUTILS_LOG_WARN(
+                    "ignoring topic '%s', which has more than one type: [%s]\n",
                     topic_name.c_str(),
-                    types.substr(0, types.length() - 2).c_str()
+                    types.str().substr(0, types.str().length() - 2).c_str()
             );
             already_ignored_topics.insert(topic_name);
           }
           continue;
         }
         if (already_ignored_topics.count(topic_name) == 0) {
           if (!check_inregex_list(whitelist_map[topic_rgxp_list_param], topic_name, valid_ros2_topics)) {
-            fprintf(
-                    stderr,
-                    "warning: ignoring topic '%s',as it does not match any regex \n",
+            RCUTILS_LOG_INFO(
```

Comment:
I think it would be useful to keep this as debug instead of info. It would be useful to know what topics are registered rather than what is ignored.